### PR TITLE
updated test case tags in all the modules

### PR DESCRIPTION
**Pull request by kbo001**
## Summary
Updated all testcase modules with new Tags 

## Description
Updated all testcase tags in all the test modules

## QA Checklist

- [X] Label(s) and Assignee added
- [X] PR sent for review
- [ ] Documentation (if applicable) added/updated

### CODE REVIEW

File: HQSmokeTests/testCases/test_01_others.py [hunk 1]
```diff
@@ -7,6 +7,15 @@
...
 @pytest.mark.smoke
+@pytest.mark.login
+@pytest.mark.dashboard
+@pytest.mark.reports
+@pytest.mark.data
+@pytest.mark.users
+@pytest.mark.applications
+@pytest.mark.webApps
+@pytest.mark.messaging
+@pytest.mark.admin
```
Comment (kiran-tripathy) [inline HQSmokeTests/testCases/test_01_others.py:18]: I don't think our user has admin rights or can see the menu.

File: HQSmokeTests/testCases/test_01_others.py [hunk 2]
```diff
@@ -7,6 +7,15 @@
 """"Contains all test cases that aren't specifically related any menu modules"""
 
 @pytest.mark.smoke
```
Comment (kiran-tripathy) [inline HQSmokeTests/testCases/test_01_others.py:9]: what's the relevance of this tag? are there non-smoke tests in this suite?

**Review by kiran-tripathy — APPROVED** [review]
left a couple of comments -- but looks good overall.

File: HQSmokeTests/testCases/test_01_others.py (hunk 1 above)
Comment (kbo001) [reply HQSmokeTests/testCases/test_01_others.py:18]: Yes, I mentioned it as it was present in the testplan. I will remove it.

File: HQSmokeTests/testCases/test_01_others.py (hunk 2 above)
Comment (kbo001) [reply HQSmokeTests/testCases/test_01_others.py:9]: I kept it thinking that we might separate the export tests from the smoke runs as they keep failing. Maybe mark the exports with a different tag and execute them separately. But now the testcases are passing so I guess its safe to remove it.

**Pull request merged**